ui/pom.xml

```diff
@@ -389,7 +389,7 @@
                                     <contextPath>/${standalone-rest-path}
                                     </contextPath>
                                     <groupId>uk.gov.gchq.gaffer</groupId>
-                                    <artifactId>accumulo-rest</artifactId>
+                                    <artifactId>map-rest</artifactId>
```

Comment:
Can you change the accumulo-store test dependency to a map-store one? It should fix the ClassNotFoundExceptions in the tests.

Reply:
Changed.

Reply:
I have checked my code as committed at the point I raised the pull request. At that point it had a dependency of map-rest rather than accumulo-rest.

It would appear that the merges from gaffer-tools:develop imposed by others have introduced problems with this. Given we have found this discrepancy I have little confidence in the rest of the code in this branch as it now stands.

Reply:
You changed it in the penultimate commit "Forced update of the branch"

Reply:
There is an accumulo-store dependency above this which should be changed to the map-store one. To verify this yourself, try running the QueryBuilderST.

Reply:
Changed the dependencies as requested.

Now getting a ui travis build failure (not failing locally).
Have you seen the error "Skipping JaCoCo execution due to missing classes directory" before?

Reply:
Nope. The exception in travis points towards missing dependencies. The sketches library is pulled in transitively by the Accumulo store dependency but seems absent in the Map Store one. Here is the stack trace from Travis CI:
```
[ERROR] uk.gov.gchq.gaffer.ui.QueryBuilderST  Time elapsed: 10.498 s  <<< ERROR!
java.lang.IllegalArgumentException: The store could not be initialised with the provided properties
	at uk.gov.gchq.gaffer.ui.QueryBuilderST.addNamedOperation(QueryBuilderST.java:416)
	at uk.gov.gchq.gaffer.ui.QueryBuilderST.beforeClass(QueryBuilderST.java:141)
Caused by: uk.gov.gchq.gaffer.store.StoreException: com.clearspring.analytics.stream.cardinality.HyperLogLogPlus (through reference chain: uk.gov.gchq.gaffer.store.schema.Builder["types"]->java.util.LinkedHashMap["hllp"]->uk.gov.gchq.gaffer.store.schema.TypeDefinition["class"])
	at uk.gov.gchq.gaffer.ui.QueryBuilderST.addNamedOperation(QueryBuilderST.java:416)
	at uk.gov.gchq.gaffer.ui.QueryBuilderST.beforeClass(QueryBuilderST.java:141)
Caused by: uk.gov.gchq.gaffer.exception.SerialisationException: com.clearspring.analytics.stream.cardinality.HyperLogLogPlus (through reference chain: uk.gov.gchq.gaffer.store.schema.Builder["types"]->java.util.LinkedHashMap["hllp"]->uk.gov.gchq.gaffer.store.schema.TypeDefinition["class"])
	at uk.gov.gchq.gaffer.ui.QueryBuilderST.addNamedOperation(QueryBuilderST.java:416)
	at uk.gov.gchq.gaffer.ui.QueryBuilderST.beforeClass(QueryBuilderST.java:141)
Caused by: com.fasterxml.jackson.databind.JsonMappingException: com.clearspring.analytics.stream.cardinality.HyperLogLogPlus (through reference chain: uk.gov.gchq.gaffer.store.schema.Builder["types"]->java.util.LinkedHashMap["hllp"]->uk.gov.gchq.gaffer.store.schema.TypeDefinition["class"])
	at uk.gov.gchq.gaffer.ui.QueryBuilderST.addNamedOperation(QueryBuilderST.java:416)
	at uk.gov.gchq.gaffer.ui.QueryBuilderST.beforeClass(QueryBuilderST.java:141)
Caused by: java.lang.ClassNotFoundException: com.clearspring.analytics.stream.cardinality.HyperLogLogPlus
	at uk.gov.gchq.gaffer.ui.QueryBuilderST.addNamedOperation(QueryBuilderST.java:416)
	at uk.gov.gchq.gaffer.ui.QueryBuilderST.beforeClass(QueryBuilderST.java:141)
```

Reply:
Added dependency as requested.
Builds OK locally.
Won't build in Travis CI due to dependency version 1.13.1 not existing.